tests/main/validate-container-happy/task.yaml

```diff
@@ -0,0 +1,38 @@
+summary: check the symlinks following the right track
+
+environment:
+    SNAP: test-snapd-validate-container-happy
+
+prepare: |
+
```

Comment:
in a followup please delete this section since there is no code being executed in it